var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -51,6 +51,10 @@ class Paraview(CMakePackage, CudaPackage):
             description='Builds a shared version of the library')
     variant('kits', default=True,
             description='Use module kits')
+    variant('cuda_arch', default='native', multi=False,
+            values=('native', 'fermi', 'kepler', 'maxwell',
+                    'pascal', 'volta', 'turing', 'all', 'none'),
```

Comment:
Is there a reason to skip `ampere`, which is supported by two of `vtk-m`'s options (`80` and `86`)?

Reply:
Hmm, I think I have looked at the code from the latest version of vtk-m and seen no mention of `ampere`: https://gitlab.kitware.com/vtk/vtk-m/-/blob/v1.5.1/CMake/VTKmDeviceAdapters.cmake#L161.

It seems to be in the master branch though so I can add it to be future-proof.

Reply:
I was just looking at the `vtk-m` package so was more curious.  I haven't investigated when the `ampere` options were added.

Reply:
IMO that can be address in a subsequent PR if it becomes an issue.

Reply:
Sure!